xml/System.Net.Sockets/Socket.xml

```diff
@@ -950,6 +950,7 @@ This method populates the <xref:System.Net.Sockets.Socket> instance with data ga
         <altmember cref="P:System.Net.Sockets.Socket.RemoteEndPoint" />
         <related type="Article" href="/dotnet/framework/network-programming/asynchronous-client-socket-example">Asynchronous Client Socket Example</related>
         <related type="Article" href="/dotnet/framework/network-programming/asynchronous-server-socket-example">Asynchronous Server Socket Example</related>
+        <exception cref="T:System.NotSupportedException">Windows NT is required for this method.</exception>
```

Comment:
Where does this come from? Afaik we didn't touch this method during the 9.0 timeframe. 

Since it looks like a legacy comment relevant for Windows variants we no longer support, it doesn't make much sense to add this.

Reply:
The tool imports any docs that were found in triple slash but not in dotnet-api-docs. It has no way of knowing if it was introduced for .NET 9 specifically or not.

If it didn't come from the `BeginAccept` method triple slash comments, then maybe it came from the parent class or the interface?

Reply:
Oh this is very weird. I am not finding this string _anywhere_ in runtime or in dotnet-api-docs.

Yes, we can remove them all.